addons/zeus/functions/fnc_addJammerZeus.sqf

```diff
@@ -36,18 +39,25 @@ private _onConfirm =
 		_unit = _dataTerminal;
 	};
 
+	private _capabilities = [];	// what types of signals can this jammer counteract?
+	if (_isVoiceCommsJammer) then { _capabilities pushBack "VoiceCommsJammer" };
+	if (_isDroneJammer) then { _capabilities pushBack "DroneJammer" };
```

Comment:
Hm don't know. 
In C/C++ I would do it using enums.
In SQF I couldn't find those.

I chose this approach because it makes the code easily readable.

If what you are suggesting is a good idea or not I can not tell. 
I simply don't feel qualified enough with SQF and it's intermingling with C++ (and how the compiler works) to judge.

Reply:
Fair enough. I'll suggest going with ``#defines`` as you are right that enums outright doesn't exist in sqf. So would make sense for define to avoid typo mistakes in future. I'll see if I can do ``suggest changes`` on this PR to directly change for that, so you just have to click accept. I'll do that when I get home from work tomorrow.